sdk/communication/Azure.Communication.Email/test-resources.json

```diff
@@ -30,12 +30,6 @@
         "metadata": {
           "description": "The application client id used to run tests."
         }
-      },
-      "testApplicationSecret": {
```

Comment:
This parameter isn't necessary and would cause the user to be prompted to input when running locally with user auth. It is not necessary for service principal auth either.